activities.json

```diff
@@ -132,6 +132,19 @@
     "title": "BigInt",
     "url": "https://tc39.github.io/proposal-bigint/"
   },
+  {
+    "ciuName": null,
+    "description": "This specification defines navigational tracking and when and how browsers are required to prevent it from happening.",
+    "id": "bounce-tracking-mitigations",
+    "mdnUrl": null,
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1839915",
+    "mozPosition": "positive",
+    "mozPositionDetail": "With 3rd-party cookies being restricted by default in all major browsers navigational tracking plays an increasingly important role on the web. This spec describes an effective cross-browser mechanism to combat bounce tracking, which does not rely on tracker lists. It provides predictable detection heuristics for web developers and preserves legitimate uses of short-lived redirects where possible.",
```

Comment:
Do you think that we should say something about the value of cross-browser alignment on behaviour?

Reply:
Good point, I've extended the comment.